dds/DCPS/RTPS/Sedp.h

```diff
@@ -209,7 +209,7 @@ class Sedp : public DCPS::EndpointManager<ParticipantData_t>, public DCPS::RcObj
 
   class MsgParticipantData : public MsgBase {
   public:
-    MsgParticipantData(DCPS::WeakRcHandle<Sedp> sedp,
+    MsgParticipantData(const DCPS::WeakRcHandle<Sedp>& sedp,
                        DCPS::MessageId id,
                        const ParticipantData_t data)
```

Comment:
This is the line that coverity didn't like

Reply:
Right.  But it seems like it will start complaining about all of the others so I might as well fix them.

Reply:
OK, but we need to update line 214 ("this is the line"... I guess GitHub doesn't make it clear) before merging this.